server/backends/disk_cache/disk_cache.go

```diff
@@ -130,6 +131,23 @@ func (c *DiskCache) Statusz(ctx context.Context) string {
 	buf += fmt.Sprintf("<div>%d items (oldest: %s)</div>", c.l.Len(), oldestItem.Format("Jan 02, 2006 15:04:05 PST"))
 	buf += fmt.Sprintf("<div>Mapped into LRU: %t</div>", *c.diskIsMapped)
 	buf += fmt.Sprintf("<div>GC Last run: %s</div>", c.lastGCTime.Format("Jan 02, 2006 15:04:05 PST"))
+
+	buckets := make([]int, 10)
```

Comment:
this'll only support files up to 10GB IIUC - maybe bump to 12 so we ppl can see if there are any extreme values (up to 1TB)?